conjur/cli.py

```diff
@@ -551,9 +551,15 @@ def run(self, *args):
             logging.debug(traceback.format_exc())
             sys.stdout.write(f"Error: No such file or directory: '{not_found_error.filename}'\n")
             sys.exit(1)
-        except requests.exceptions.HTTPError as client_error:
+        except requests.exceptions.HTTPError as server_error:
             logging.debug(traceback.format_exc())
-            sys.stdout.write(f"Failed to execute command. Reason: {client_error}\n")
+            # pylint: disable=no-member
+            if hasattr(server_error.response, 'status_code') and str(server_error.response.status_code) == '401':
+                sys.stdout.write("Failed to log in to Conjur. Unable to authenticate with Conjur. "
+                    f"Reason: {server_error}. Check your credentials and try again.\n")
```

Comment:
what we do here is forwording the http error message with some cli instructions 
I think not only 401 error is relevant but more (almost all 500's could be followed with some general instructions) so I think we should extract the handle to outer function where we could add handeling for other errors over time.
BTW CLI can be very sophisticate and for every error it could narrow down the actual problem and give the user very good and informative instructions 

```suggestion
            if hasattr(server_error.response, 'status_code') #This should check if we can filter by error code
                print_user_informative_error(server_error)
        def print_user_informative_error(server_error):
            if str(server_error.response.status_code) == '401':
                   sys.stdout.write("Failed to log in to Conjur. Unable to authenticate with Conjur. "
                       f"Reason: {server_error}. Check your credentials and try again.\n")
```

Reply:
"BTW CLI can be very sophisticate and for every error it could narrow down the actual problem and give the user very good and informative instructions"
how do you suggest?

Reply:
for some http error we can give a set of instruction.
for example if it is a 502 we can tell the user to check if conjur is up. if we want to be even more sophisticate we can check for server /info and if it is a standby we can raise "note that you are trying to access a standby machine"